lib/api-fallback.ts

```
@@ -0,0 +1,87 @@
+  static getStatus(): {
+    usingEdge: boolean
+    failureCount: number
+    lastFailureTime: number
+  } {
+    return {
+      usingEdge: this.shouldUseEdgeFunctions(),
+      failureCount: this.failureCount,
+      lastFailureTime: this.lastFailureTime
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)